discovery-infra/tests/test_targets.py

```diff
@@ -10,3 +14,19 @@ def test_target_deploy_nodes(self, cluster):
     @JunitTestSuite()
     def test_target_install_with_deploy_nodes(self, prepared_cluster):
         prepared_cluster.start_install_and_wait_for_installed()
+
+    # Day 2 #
+    @pytest.fixture
+    def new_cluster_configuration(self):
+        return Day2.override_cluster_configuration()
+
+    @JunitTestSuite()
+    def test_target_deploy_day2_nodes_cloud(self, cluster):
+        # Deploy day2 nodes on an installed cluster (should specify CLUSTER_ID env var)
+        Day2().test_deploy_nodes_cloud(cluster)
+
+    @JunitTestSuite()
+    def test_target_install_with_deploy_day2_nodes_cloud(self, prepared_cluster):
+        # Install day1 cluster and deploy day2 nodes (cloud flow)
+        self.test_target_install_with_deploy_nodes(prepared_cluster)
+        Day2().test_deploy_nodes_cloud(prepared_cluster)
```

Comment:
shouldn't it work with ``make test TEST=./discovery-infra/tests/test_day2.py``?
I think it's better to not duplicate it